### DISCUSSION TITLE
And the Hacktoberfest Main Prize Winner is...

### DISCUSSION THREAD

**Discussion by C0staTin** [mindsdb/mindsdb#1756] [Announcements]
This year's Hacktoberfest was great!
We wanted to thank everyone for all the submissions to both the [MindsDB](https://github.com/mindsdb/mindsdb/) and [Lightwood](https://github.com/mindsdb/lightwood) repos.
You made 584 comments, 309 open pull requests, 191 merge pull requests, opened 179 issues, and added 142 stars across all repos. The audio encoder, quantum mixers, and many new database connectors are also much appreciated!
The video of the prize draw is posted at https://mindsdb.com/hacktoberfest/
Please check it to see who WON the Deep Learning Laptop! 
Thanks for contributing to MindsDB! :rocket: (edited)

**Comment by MichaelLantz** [comment]
Congratulations Kamil on the draw! Another congratulations to MindsDB on the new seed funding to help propel Machine Learning forward.

**Reply by C0staTin** [reply]
Thanks, Michael! 
This is actually the greatest Hacktoberfest for MindsDB.
And it would not be possible without contributors like you. You did exceptional work!
Thank you!

**Comment by C0staTin** [comment]
I don't want to spoil much here, but we are working on keeping the momentum and are building smth to encourage great contributors.
So stay tuned!
